dataprofiler/tests/profilers/test_float_column_profile.py

```diff
@@ -580,12 +633,12 @@ def test_profile(self):
                 'bin_edges': np.array([2.5, 5.0, 7.5, 10.0, 12.5]),
             },
             quantiles={
-                0: 3.75,
-                1: 5.0,
-                2: 10.0
+                0: 2.5075,
+                1: 5.005 ,
+                2: 12.4925,
             },
```

Comment:
Could we merge to the nearest bin? Or is that inaccurate?

Reply:
we don't know where in the bin it exists so we choose an interpolation. If we know it's integer, we could to the nearest half. Additionally, quantiles aren't only at 25,50,75. That's just the current test, not that we want to someone could technically ask for 49.9999.

Reply:
Recommend switching to at least this: https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertAlmostEqual

Reply:
We could but then we could have a slight change in quantiles which we would miss for and it would be inaccurate. For instance if the test chose 248 instead of 249 it might pass